docs/source/reservations/reservations.rst

```diff
@@ -77,16 +84,18 @@ For example, if you are launching a cluster with the following SkyPilot YAML:
     num_nodes: 2
 
 
-SkyPilot will utilize the capacity reservation as follows:
+SkyPilot will utilize the capacity reservation/block as follows:
 
-1. Find available capacity in your capacity blocks in ``us-east-2`` and ``us-west-2`` in the reservation ``cr-123456789a`` and ``cr-23456789ab``, respectively. Let's say 1 A100 instance capacity is available in ``us-east-2`` but no available capacity in ``us-west-2`` (as shown in the figure above).
-2. SkyPilot will first try to launch 2 nodes in ``us-east-2`` with 1 node coming from the reservation, and the rest 1 node as on-demand.
-3. If it fails to find available an on-demand A100 instance in ``us-east-2``, it will automatically :ref:`failover <auto-failover>` to other clouds/regions/zones.
+1. Query reservations/blocks in ``us-east-2`` and ``us-west-2`` in reservation ``cr-123456789a`` and ``cr-23456789ab``, respectively. Let's say:
+   - 1 A100 instance capacity is available in ``us-east-2``,
+   - No available capacity in ``us-west-2``.
+2. SkyPilot calculates the pricing for all zones and finds that zones in ``us-east-2`` are cheaper than other zones because it is considered as 1 on-demand price for 2 nodes.
```

Comment:
```suggestion
2. SkyPilot calculates the pricing for all zones as described above.  The result is ``us-east-2`` zones are cheaper than  all other zones, because the former's costs are 1 on-demand node's cost for 2 nodes (by satisfying 1 node using the reserved capacity).
```